devops/ansible/playbook.yml

```diff
@@ -40,12 +40,12 @@
     apt: pkg=mongodb-org state=present update_cache=yes
     sudo: yes
 
-  - name: install pycurl package in order to use the apt_repository module
-    apt: pkg=python-pycurl state=present
+  - name: install curl so we can install nodejs
+    apt: pkg=curl state=present
     sudo: yes
 
   - name: add nodejs ppa
-    apt_repository: repo='ppa:chris-lea/node.js'
+    shell: curl -sL https://deb.nodesource.com/setup | sudo bash -
     sudo: yes
 
   - name: update apt cache
```

Comment:
The script is essentially a black box to us, we aren't really guaranteed that it will perform the apt cache update, unless the node people explicitly guarantee that they will always do that in their script. Absent such a guarantee, I think I will leave this here and go ahead and merge it. Thanks for the review :smile: